### DISCUSSION TITLE
How i can change color of blankline?

### DISCUSSION THREAD

**Discussion by KirovAI** [NvChad/NvChad#1390] [Q&A] [answered]
default colors of blankline(grey and white) merge with bg on small displays. Maybe possible change their to 
white blankline if it unactive and purple if active?

**Comment by siduck** [comment, marked as answer]
check blankline highlights in base46 repo's integration dir and read the docs to override highlight groups

**Comment by KirovAI** [comment]
I get bugs after rewriting blackline colors. Highlighting do not take effect after I enable Insert mode. Video:


https://user-images.githubusercontent.com/105173528/182229907-f6503187-4419-4741-86ee-31c0d1f339ca.mp4



My config:
https://pastebin.com/vkdYNa0v

**Reply by siduck** [reply]
@KirovAI as you are overriding default highlight group, you must use put it in `hl_override`,  

`hl_add` is for adding totally new highlight groups that are not defined by nvchad. Read the docs next time!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
